src/tailscale_svc_lb_controller/config.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import logging
+import re
+import sys
+from os import environ as env
+
+# -- Constants
+CONTROLLER_PREFIX = "svc-lb.tailscale.iptables.sh"
+SERVICE_NAME_LABEL = CONTROLLER_PREFIX + "/svc-name"
+SERVICE_NAMESPACE_LABEL = CONTROLLER_PREFIX + "/svc-namespace"
+
+# -- Proxy Deployment Configuration
+RESOURCE_PREFIX = env.get("RESOURCE_PREFIX", "ts-")
+SECRET_NAME = env.get("SECRET_NAME", "tailscale-svc-lb")
+# LoadBalancerClass this instance of Tailscale Proxy will implement
+LOAD_BALANCER_CLASS = env.get("LOAD_BALANCER_CLASS", CONTROLLER_PREFIX + "/lb")
+# Label to use when selecting nodes for the Tailscale Proxy to run on. The value of this label should be 'true'
+NODE_SELECTOR_LABEL = env.get("TS_PROXY_NODE_SELECTOR_LABEL", None)
+
+# A semi-colon seperated string containing the names of any secrets that should be used
+# when pulling images. Secret must already exist and be present in the TS_PROXY_NAMESPACE
+IMAGE_PULL_SECRETS = env.get("IMAGE_PULL_SECRETS", "")
+if not re.match(r"^([a-z]|-|\d|;)*$", IMAGE_PULL_SECRETS):
+    logging.error("IMAGE_PULL_SECRETS invalid. Should be a semi-colon seperated list of"
+                  "secret names.")
+    sys.exit(1)
+
+# Type of deployment to use for the Tailscale Proxy instances
+DEPLOYMENT_TYPE = env.get("DEPLOYMENT_TYPE", "DaemonSet")
+if DEPLOYMENT_TYPE not in ["DaemonSet", "Deployment"]:
+    logging.error("DEPLOYMENT_TYPE invalid. Valid options are 'DaemonSet', 'Deployment'")
+    sys.exit(1)
+
+# Tailscale Proxy Runtime Namepace. All Tailscale Proxies will be created in this namespace.
+TS_PROXY_NAMESPACE = env.get("TS_PROXY_NAMESPACE", "default")
+
+# If TS_PROXY_DEPLOYMENT_TYPE is 'Deployment', this dictates the number of replicas. No effect otherwise.
+try:
+    TS_PROXY_REPLICA_COUNT = int(env.get("TS_PROXY_REPLICA_COUNT", "2"))
+except Exception:
```

Comment:
Should catch specific error here:
```suggestion
except ValueError:
```